src/commands/accesslogs.js

```diff
@@ -0,0 +1,67 @@
+'use strict';
+
+const AppConfig = require('../models/app_configuration.js');
+const User = require('../models/user.js');
+const Logger = require('../logger.js');
+const {
+  getToken,
+  AccessLogsBatchEmitter,
+  getAccessLogsFromWarp10InBatches
+} = require('../clever-client/access-logs.js');
+const moment = require ('moment');
+
+async function accessLogs (params) {
+  const { alias, format, before, after } = params.options;
+
+  const { org_id: orgaId, app_id: appId } = await AppConfig.getAppData(alias).toPromise();
+  const userId = await User.getCurrentId();
+  const ownerId = orgaId || userId;
+
+  let from;
+  let to;
+  if(before == null && after == null) {
+    from = moment().subtract(1, 'hour');
+    to = moment();
+  } else if(before == null && after != null) {
+    from = moment(after);
+    to = moment();
+  } else if(before != null && after == null) {
+    from = moment(before).subtract(1, 'hour');
+    to = moment(before);
+  } else {
+    from = moment(after);
+    to = moment(before);
+  }
+
+  const warpToken = await getToken({ orgaId: ownerId });
+  const emitter = new AccessLogsBatchEmitter();
+  getAccessLogsFromWarp10InBatches({ ownerId: orgaId, appId, before: to, after: from, warpToken, emitter });
```

Comment:
I am. We already talked about this elsewhere, I am completely open to using something else. I only used moment because I saw that it was already used elsewhere (and saw no replacement).